presto-main/src/main/java/com/facebook/presto/server/InternalCommunicationModule.java

```diff
@@ -13,6 +13,7 @@
  */
 package com.facebook.presto.server;
 
+import com.facebook.presto.util.KerberosPrincipal;
```

Comment:
If you build the principal name based on the service name you will no longer need this commit. Feel free to drop it.